less/admin/UsersListPage.less

```diff
@@ -0,0 +1,118 @@
+.UserListPage {
+  // Pad bottom of page to make nav area look less squashed
+  margin-bottom: 24px;
+
+  &-grid {
+    width: 100%;
+    position: relative;
+    border-radius: @border-radius;
+
+    // Ensure mobile scrollbar isn't on top of content
+    padding-bottom: 4px;
+
+    // Table refreshing overlay
+    &--loadingPage {
+      &::after {
+        content: "";
+        display: block;
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        background: rgba(128, 128, 128, 0.2);
+      }
+    }
+
+    .LoadingIndicator {
+      display: block;
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      left: 0;
+      right: 0;
+    }
+
+    &--loaded,
+    &--loadingPage {
+      display: grid;
+      overflow-x: auto;
+    }
+
+    &--header {
+      font-weight: bold;
+      border-bottom: 1px solid @muted-more-color;
+      padding: 8px;
+      background: @control-bg;
+    }
+
+    &--rowItem {
+      padding: 4px 8px;
+    }
+
+    &--shadedRow {
+      background: darken(@body-bg, 3%);
+
+      & when (@config-dark-mode = true) {
+        background: lighten(@body-bg, 5%);
+      }
+    }
+  }
+
+  &-gridPagination {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 16px;
+
+    button .Button-icon {
+      margin-right: 0;
+    }
```

Comment:
let's remove this and use the `Button--icon` class on those arrow buttons instead.